### PR TITLE
[ENHANCEMENT] updated default run_name in notebook to be a timestamp

### PR DESCRIPTION
**Pull request by talagluck**
Changes proposed in this pull request:
- Changed the default run_name in the suite edit notebook to be a time stamp
- Changed the default run_name in the suite scaffold notebook to be a time stamp

The strftime conversion to string in the notebook feels feels a bit inelegant - I ran into issues with the action list validation operator when just using a datetime object so I went with that, but am open to other suggestions.

### CODE REVIEW

**Review by Aylr — CHANGES_REQUESTED** [review]
While we are here, I'm partial to removing this entire chunk of docs and run id so it gets a run time by default. It's less for the user to read and the point here isn't education.

**Comment by talagluck** [comment]
@Aylr I removed the run_id, but force-pushed so I missed your other feedback. What else does this PR need?

File: great_expectations/render/notebook_assets/suite_edit/footer.py.j2 [hunk 1]
```diff
@@ -1,20 +1,6 @@
...
-arguments are given, run_name will be None and run_time will default to the current UTC datetime.
-"""
-
-run_id = {
-  # insert your own run_name if you do not want a timestamp
-  "run_name": datetime.datetime.now(datetime.timezone.utc).strftime("%Y%m%dT%H%M%S.%fZ"),
-  "run_time": datetime.datetime.now(datetime.timezone.utc)
-}
-
-results = context.run_validation_operator("action_list_operator", assets_to_validate=[batch], run_id=run_id)
```
Comment (Aylr) [inline great_expectations/render/notebook_assets/suite_edit/footer.py.j2]: ❤️ ❤️

**Review by Aylr — APPROVED** [review]
I love this. I've seen multiple users get lost in details in the suite edit notebook and this is a big step forward in simplicity!

**Pull request merged**